config/config.go

```diff
@@ -49,19 +49,29 @@ func (g Generator) ParamString() string {
 }
 
 func (g Generator) ParamStringWithOut() string {
+	// If no out path was specified, use the package directory.
 	outPath := g.OutPath()
+	if outPath == "" {
+		outPath = g.ConfigDir
+	}
```

Comment:
But the config dir isn't always the package dir :)

Reply:
Hmm, I guess we will have to pass in the gunk package path, and use that rather than `g.ConfigDir`

Reply:
> I guess we will have to pass in the gunk package path

assuming you mean the directory, not the import path, it sounds like an option :)

Reply:
Yes I am :) Done.